src/Raven.Server/Integrations/PostgreSQL/PgSession.cs

```diff
@@ -273,5 +278,22 @@ private string GetSourceConnectionDetails(string userName)
 
             return details;
         }
+        
+        private void DispatchPostgresQueryMessageToTrafficWatch(Query message, string nodeTag)
+        {
+            var clientIp = _client.Client.LocalEndPoint?.ToString();
+            string databaseName = _clientOptions.GetValueOrDefault("database", "N/A");
+
+            var twn = new TrafficWatchPostgresChange()
+            {
+                TimeStamp = DateTime.UtcNow,
+                DatabaseName = databaseName,  
+                CertificateThumbprint = _serverCertificateHolder.Certificate?.Thumbprint,
```

Comment:
this is not relevant in here, should be set to null?

Reply:
Do we want to put the postgres username here somewhere?